src/docs/swagger.js

```diff
@@ -497,6 +501,39 @@ export default {
         ]
       }
     },
+    '/articles/{slug}/statistics?year={year}&month={month}': {
+      get: {
+        summary: 'Get reading statistics of an article',
+        description: '',
+
+        consumes: ['application/x-www-form-urlencoded'],
+        responses: {
+          200: {
```

Comment:
Why is the documentation lacking for the structure of the response which comes back on success?

Reply:
Fixed.